fixtures/fixtures.js

```diff
@@ -1,6 +1,41 @@
-import Route from 'route-parser';
-
+import Store from '../src/store.js';
 var Fixtures = {
+    userTraders: {
```

Comment:
But technically speaking - according to our data - they are just trader objects. But I guess if you are just implying that this is a list of the traders attached to this user which has already been filtered by the API then maybe it makes sense.

Not going to fight you on it.